.env.example

```diff
@@ -1,7 +0,0 @@
-DB_CONNECTION_URL=From MongoDB Atlas Panel > CONNECT > Connect Your Application > Connection String
-USE_DOCKER= true
-MONGO_INITDB_ROOT_USERNAME= admin1
-MONGO_INITDB_ROOT_PASSWORD= adminPassword
-MONGO_INITDB_DATABASE= RESTfetchdb
-MONGO_NON_ROOT_USERNAME= user1
-MONGO_NON_ROOT_PASSWORD= userPassword
```

Comment:
Why was this file deleted?

Reply:
> Why was this file deleted?

It was deleted by mistake since .env was already made by me, i didnt notice this one. My apologies!
I am trying if i can add this file again in this PR

Reply:
> > Why was this file deleted?
> 
> It was deleted by mistake since .env was already made by me, i didnt notice this one. My apologies!
> I am trying if i can add this file again in this PR

I have added the file by copy pasting from original repo, will that work